packages/dashboard/src/components/schedule-visualizer/index.tsx

```diff
@@ -43,6 +44,7 @@ export interface MapFloorLayer {
 export interface ScheduleVisualizerProps extends React.PropsWithChildren<{}> {
   buildingMap: RomiCore.BuildingMap;
   negotiationTrajStore: Readonly<Record<string, NegotiationTrajectoryResponse>>;
+  omniPanelView: number;
```

Comment:
This coupled it with the omnipanel, instead, leave the logic whether to show/hide trajectories to `Dashboard`, which has knowledge of both `OmniPanel` and `ScheduleVisualizer` to make that decision.

Reply:
Used a boolean check at the dashboard instead